### ISSUE TITLE
Gradient-Regulated Meta-Prompt Learning for Generalizable Vision-Language Models

### ISSUE THREAD

**Issue by sandys** [arakoodev/EdgeChains#92]
https://arxiv.org/abs/2303.06571

**Comment by auniquesun** [comment]
@sandys what do you mean by sharing this link? The current repository seems not corresponding the paper you mentioned. I am also searching the code of that paper. How about your progress?

**Comment by sandys** [comment]
> @sandys what do you mean by sharing this link? The current repository seems not corresponding the paper you mentioned. I am also searching the code of that paper. How about your progress?

hi,
thanks for commenting here. the issue is created because this is a work that a volunteer will take up. it is not yet incorporated into the codebase. 
are u interested in this particular algorithm ?

**Comment by auniquesun** [comment]
@sandys yes, I am interested in the algorithm. I want to learn the implementation but there is no released code.

**Comment by qttis** [comment]
> Refe

me too.
